src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Appearance/ApplicationAccentColorManager.cs

```diff
@@ -0,0 +1,317 @@
+            .Diagnostics
+            .Debug
+            .WriteLine("INFO | SystemAccentColorTertiary: " + tertiaryAccent, "System.Windows.Accent");
+#endif
+
+        if (secondaryAccent.GetBrightness() > BackgroundBrightnessThresholdValue)
+        {
+#if DEBUG
+            System.Diagnostics.Debug.WriteLine("INFO | Text on accent is DARK", "System.Windows.Accent");
+#endif
+            Application.Current.Resources["TextOnAccentFillColorPrimary"] = Color.FromArgb(
+                0xFF,
+                0x00,
+                0x00,
+                0x00
+            );
+            Application.Current.Resources["TextOnAccentFillColorSecondary"] = Color.FromArgb(
+                0x80,
+                0x00,
+                0x00,
+                0x00
+            );
+            Application.Current.Resources["TextOnAccentFillColorDisabled"] = Color.FromArgb(
+                0x77,
+                0x00,
+                0x00,
+                0x00
+            );
+            Application.Current.Resources["TextOnAccentFillColorSelectedText"] = Color.FromArgb(
+                0x00,
+                0x00,
+                0x00,
+                0x00
+            );
+            Application.Current.Resources["AccentTextFillColorDisabled"] = Color.FromArgb(
+                0x5D,
+                0x00,
+                0x00,
+                0x00
+            );
+        }
+        else
+        {
+#if DEBUG
+            System.Diagnostics.Debug.WriteLine("INFO | Text on accent is LIGHT", "System.Windows.Accent");
+#endif
+            Application.Current.Resources["TextOnAccentFillColorPrimary"] = Color.FromArgb(
+                0xFF,
+                0xFF,
+                0xFF,
+                0xFF
+            );
+            Application.Current.Resources["TextOnAccentFillColorSecondary"] = Color.FromArgb(
+                0x80,
+                0xFF,
+                0xFF,
+                0xFF
+            );
+            Application.Current.Resources["TextOnAccentFillColorDisabled"] = Color.FromArgb(
+                0x87,
+                0xFF,
+                0xFF,
+                0xFF
+            );
+            Application.Current.Resources["TextOnAccentFillColorSelectedText"] = Color.FromArgb(
+                0xFF,
+                0xFF,
+                0xFF,
+                0xFF
+            );
+            Application.Current.Resources["AccentTextFillColorDisabled"] = Color.FromArgb(
+                0x5D,
+                0xFF,
+                0xFF,
+                0xFF
+            );
+        }
+
+        Application.Current.Resources["SystemAccentColor"] = systemAccent;
+        Application.Current.Resources["SystemAccentColorPrimary"] = primaryAccent;
+        Application.Current.Resources["SystemAccentColorSecondary"] = secondaryAccent;
+        Application.Current.Resources["SystemAccentColorTertiary"] = tertiaryAccent;
+
+        Application.Current.Resources["SystemAccentBrush"] = secondaryAccent.ToBrush();
```

Comment:
Can I know why do we not use `systemAccent.ToBrush()` to create the `SystemAccentBrush` ?

```suggestion
        Application.Current.Resources["SystemAccentBrush"] = systemAccent.ToBrush();
```

Reply:
Looks like an oversight. Fixed it now.
Thanks